about.html

```diff
@@ -0,0 +1,130 @@
+            <h3 class="pt-5 pb-2 mt-2" style="color: rgb(255, 51, 0); font-weight: 400;font-size: 30px;">"Hello, Sharing
+                world"</h3>
+            <h1 style="color: rgb(255, 51, 0);">CONTAINER FLUID</h1>
+            <h1 style="color: rgb(255, 51, 0);">ARTS FESTIVAL</h1>
+            <h1 style="color: rgb(255, 51, 0);">2022</h1>
+            <p class="bg-light my-3 p-1" style="font-size: 18px;width: 100%;">The mission of the Container Festival is to
+                promote the arts and enhance the
+                quality of life through the production of an annual outdoor, multi-disciplinary event. We strive to
+                maintain the highest artistic quality in our production and programming, represent excellence among a
+                range of artistic mediums, and promote an appreciation for fine art while encouraging innovation in the
+                field, and supporting non-traditional, contemporary works</p>
+        </div>
+        <a class="logo text-center" style="margin-bottom:5%;" href=""><img src="img/logo.png" alt="logo creative commons"></a>
+
+        <h1 class="text-center fs-3 " style="color: rgb(0, 0, 0);">Our Activities</h1>
+        <hr class='mb-5'
+            style="color:rgb(255, 51, 0,1) ;border: 2px solid rgb(255, 51, 0,1);width: 25%; margin-left: 37%;">
+            <p class="text-center">There are two ocasions where the Container Festival 
+                came to agreement and constructed more spaces for culture</p>
+
+        <div class="imgwrapper justify-content-center">
+            <div class="singleimg">
+                <div class="imgabout p-0" style="width: 100%;">
+                    <img src="img/aboutimage.jpg" alt="about image 1" width="400" height="300">
+    
+                    <div class="redmask">
+                        <div>
+                            <h6 class='ps-4' style="color: rgb(255, 255, 255);font-weight: lighter;">
+                                3rd January 2023
+                            </h6>
+                            <p style="color: white;font-weight: bolder;font-size: 26px;">Bogota Arena</p>
+                        </div>
+                    </div>
+    
+                </div>
+            </div>
+
+            <div class="singleimg">
+                <div class="imgabout p-0" style="width: 100%;">
+                    <img src="img/aboutimage3.jpg" alt="about image 2" width="400" height="300">
+    
+                    <div class="redmask">
+                        <div>
+                            <h6 class='ps-5' style="color: white;">
+                                2nd June 2023
+                            </h6>
+                            <p style="color: white;font-weight: bolder;font-size: 26px;">Medellin Arena</p>
+                        </div>
+                    </div>
+    
+                </div>
+            </div>
+        </div>
+
+            <img class="aboutlogo" src="img/cc-logo2.png" alt="logo creative commons">
```

Comment:
- [ ] Great job on not making the partners section for the desktop version however, it's present and visible for the mobile version, kindly add it 👍  
- [ ] Kindly try to follow design guidelines for the footer in the about page. for desktop version it's dark and for mobile it's light 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198696134-40415209-698c-45c0-8f69-fd06627c95f6.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/198696168-aa3dbf0f-cb2d-471e-a9a4-a38a072ddff6.png)